tools/mo/openvino/tools/mo/ops/BatchNormInference.py

```diff
@@ -3,7 +3,7 @@
 
 from openvino.tools.mo.graph.graph import Graph
 from openvino.tools.mo.ops.op import Op
-
+from openvino.tools.mo.front.common.partial_infer.utils import reverse_bypass_infer
```

Comment:
please sort imports and check other files

Reply:
done